options:
  control_mode: "torque"
  is_optitrack_on: true
  filter_gain: 0.2

control:
    dsGain: 1.
